<launch>

	<!-- XBox controller node -->
	<node name="osa_joy_node" pkg="joy" type="joy_node" respawn="true">
		<param name="dev" type="string" value="/dev/input/js0" />
		<remap from="joy" to="joy0"/>
	</node>
	
	<!-- Tank Drive node -->
	<node name="osa_tank_drive_node" pkg="osa_control" type="osa_tank_drive" respawn="true">
		
		<!-- These parameters are the 2 DOF of the wheels to control in a tank drive style -->
		<param name="dof_right_wheel" type="string" value="dof17" />
		<param name="dof_left_wheel" type="string" value="dof18" />
		
		<!-- These parameters are the indexes in the axes array of a std_msgs::Joy -->
		<param name="joy_axis_left_right" type="int" value="3" />
		<param name="joy_axis_up_down" type="int" value="4" />
		
		<!-- Joystick topic to listen -->
		<remap from="joy" to="joy0"/>
	</node>
	
</launch>
